src/app/day-calendar/day-calendar-config.model.ts

```diff
@@ -18,8 +18,10 @@ export interface IDayCalendarConfig extends ICalendar {
   yearFormatter?: (year: Moment) => string;
   dayBtnFormat?: string;
   dayBtnFormatter?: (day: Moment) => string;
+  dayBtnCssClass?: (day: Moment) => string;
```

Comment:
One might want to add multiple classes, will it work?

Reply:
Yes it will - how can I make this clear in the README>

Reply:
just add this information/example in the configuration description.